src/nb_type.cpp

```diff
@@ -40,6 +40,72 @@ static PyObject **nb_weaklist_ptr(PyObject *self) {
     return weaklistoffset ? (PyObject **) ((uint8_t *) self + weaklistoffset) : nullptr;
 }
 
+static void nb_enable_try_inc_ref(PyObject *obj) noexcept {
+#if 0 && defined(Py_GIL_DISABLED) && PY_VERSION_HEX >= 0x030E00A5
+    PyUnstable_EnableTryIncRef(obj);
+#elif defined(Py_GIL_DISABLED)
+    // TODO: Replace with PyUnstable_Object_EnableTryIncRef when available.
```

Comment:
What's the difference between `PyUnstable_Object_EnableTryIncRef` and `PyUnstable_EnableTryIncRef`?

Reply:
No difference, just the name changed. I'll update the comment.